frontend/packages/ceph-storage-plugin/src/utils/kebab-actions.ts

```diff
@@ -0,0 +1,19 @@
+import { DeleteSnapshot } from './delete-snapshot-workflow';
+import { K8sKind } from '@console/internal/module/k8s';
+import { KebabAction } from '@console/internal/components/utils';
+import { PersistentVolumeClaimModel } from '@console/internal/models/index';
+import { RestorePVC } from './restore-pvc-workflow';
+import { SnapshotPVC } from './snapshot-workflow';
+import { VolumeSnapshotModel } from '../models';
+
+export const getKebabActionsForKind = (resourceKind: K8sKind): KebabAction[] => {
+  if (resourceKind) {
+    if (resourceKind.kind === PersistentVolumeClaimModel.kind) {
```

Comment:
We need to be checking API groups as well

Reply:
@cloudbehl and @gnehapk are working on a separate PR to add extensions (OCS service and provisioner based). (This file has been pulled from https://github.com/openshift/console/pull/3927)